### ISSUE TITLE
ClickUrl dont have effekt.

### ISSUE THREAD

**Issue by a-t-k** [enkelmedia/TheDashboard#103]
Version 14.0.0

DashboardCounterModel has property "ClickUrl".
But this is not rendered in HTLM and so no click exists in dashboard.

in TS

```
<div>
          ${when(this.counters,()=>html`
            ${repeat(this.counters!.counters,
              (item)=>item.text,
              (counter)=>html`
                <div class="counter">
                  <div>
                    <span class="dot ${counter.style}">${counter.count}</span>
                  </div>
                  <div>
                    <p>
                      ${when(counter.localizationKey,()=>html`${this.localize.term(counter.localizationKey)}`)}
                      ${when(!counter.localizationKey,()=>html`${counter.text}`)}
                    </p>
                  </div>
                </div>
            `)}
          `)}
</div>
```

Is this intentional or was it forgotten to be implemented?

**Comment by enkelmedia** [comment]
Ohh... I remember that there was "something" with this but I can't recall the details. It's for sure missing the link. Let me investigate this a little further and get back.

**Comment by enkelmedia** [comment]
Hi!

Thanks a lot for reporting this, it has been fixed in `15.0.1` and `14.0.1`.

I remember that I was thinking about a more flexible way to define what should happen when someone clicks, e.g. being able to open a modal or similar stuff. But until I've got this figured out the old `clickUrl` will hopefully solve most situations.
